xml/operations-configuring-identity-websso.xml

```diff
@@ -840,4 +855,250 @@ keystone_sp_conf:
    dashboard service with your ADFS credentials.
   </para>
  </section>
+ </section>
+ <section>
+  <title>WebSSO Using OpenID Method</title>
+   <para>
+    The interaction between Keystone and external Identity Provider (IDP) is
+    handled by the Apache2 <link xlink:href="https://github.com/zmartzone/mod_auth_openidc">auth_mod_openidc</link> module.
+   </para>
+   <para>
+     There are two steps to enable to feature.
+   </para>
+   <orderedlist>
+    <listitem>
+     <para>
+      Configure Keystone with the required OpenID Connect provider information.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Create the Identity Provider, protocol, and mapping in Keystone, using
+      <emphasis role="bold"> OpenStack Command Line Tool</emphasis>.
+     </para>
+    </listitem>
+   </orderedlist>
+  <section>
+  <title>Configuring Keystone</title>
+   <orderedlist>
+    <listitem>
+     <para>
+      Log in to the &clm; node and edit the <literal>~/openstack/my_cloud/config/keystone/keystone_deploy_config.yml</literal> file with the <literal>"keystone_openid_connect_conf"</literal> variable. For example:
+     </para>
+<screen>keystone_openid_connect_conf:
+    identity_provider: google
+    response_type: id_token
+    scope: "openid email profile"
+    metadata_url: https://accounts.google.com/.well-known/openid-configuration
+    client_id: 19853478932-226lc04this22f8mc3iloi3f1rk4n5h7.apps.googleusercontent.com
+    client_secret: KmO4YwdXYZOptTTYwOiR-6d9
```

Comment:
Rather than giving an example secret which some customers might actually use I'd rather show something like this:

  client_secret: <Replace with your client secret"

Reply:
Done